README.md

```diff
@@ -96,57 +113,60 @@ in your own QML files.
 - declare the tool you wish to use. Generally you will also have to supply the
   `GeoView` etc. you wish the tool to work with. For example, to add a `NorthArrow`:
 
-```
-import esri.arcgisruntime.toolkit 1.0
-...
-NorthArrow {
+```qml
+import Esri.ArcGISRuntime 100.X
+import "qrc:///Esri/ArcGISRuntime/Toolkit" as Toolkit
+
+MapView {
+    id: mapView
+
+    Map {
+      BasemapTopographic {}
+    }
+
+    Toolkit.NorthArrow {
         id: arrow
         width: 32
         height: 32
         anchors {
             right: parent.right
             bottom: view.bottom
         }
-
-        mapView: view
+        geoView: mapView
     }
+}
 ```
 
-### toolkitwidgets
+### toolkitwidgets.pri
+
+#### importing the toolkit (toolkitwidgets.pri)
 
-#### importing the toolkit
 A good way to start using the toolkit is to use one of the template apps which
 get added to QtCreator when you install the ArcGIS Runtime SDK for Qt.
 
-- In QtCreator choose `File/New file or project/ArcGIS/ArcGIS Runtime 100.6 Qt
+- In QtCreator choose `File/New file or project/ArcGIS/ArcGIS Runtime 100.X Qt
   Widgets App
 - choose settings to match the platform etc. you are building for
 - in the .pro` file of your new app, add a line to add the library for your QML
   plugin - for example:
 
-```
-include(path/to/toolkit/toolkitwidgets.pri)
+```pro
+include(path/to/toolkitwidgets.pri)
 ```
 
-#### using a tool from the toolkit
+#### using a tool from the toolkit (toolkitwidgets.pri)
+
 Once you have successfully imported the toolkit, you can create individual tools
 in your own widgets files.
 
 - in your widgets code, create a new instance of the tool you wish to use and
-  add it your widets layout. You will also normally need to pass the `GeoView`
+  add it your widgets layout. You will also normally need to pass the `GeoView`
   which the tool is designed to work with:
 
-```
-  #include "NorthArrow.h"
+```cpp
+  #include "Esri/ArcGISRuntime/Toolkit/NorthArrow.h"
   ...
   auto northArrow = new NorthArrow(this);
-  northArrow->setGeometry(0,0,48,48); // set size and position for the tool
   northArrow->setMapView(m_mapView);
```

Comment:
is this not `setGeoView`?

Reply:
In the widgets I elected for two: `setMapView` and `setSceneView` functions so there was only ever one _type_ of View that was allowed in those cases. Not sure if good or bad, but in the widget case `setMapView` is correct.